plugins/NitroSpoof/src/main/java/com/xinto/aliuplugins/nitrospoof/Util.kt

```diff
@@ -1,4 +1,9 @@
 package com.xinto.aliuplugins.nitrospoof
 
+import com.aliucord.Constants.ALIUCORD_GUILD_ID
+
 const val EMOTE_SIZE_DEFAULT = "48"
 const val EMOTE_SIZE_KEY = "emote_size"
+const val EMPTY_CHAR = "[‎ ]"
+val permBlacklist = mapOf(ALIUCORD_GUILD_ID to "Aliucord")
+val servBlacklist = mutableMapOf(1015931589865246730 to "Vendetta")
```

Comment:
Why is this a separate list? Also why is the Vendetta server blacklisted?

Reply:
permBlacklist is unmutable, as Aliucord won't allow this ever.
the servBlacklist is mutable, this is what we modify with commands

Reply:
Understandable, but what did Vendetta do?

Reply:
they didnt allow invisible characters too so

Reply:
Ah, understood.